Bigtable/VERSION

```diff
@@ -1 +1 @@
-0.5.0
\ No newline at end of file
+0.6.0
```

Comment:
#1136 was against https://github.com/GoogleCloudPlatform/google-cloud-php/tree/bigtable. We don't need to add it to the release.

Reply:
Done